spec/models/work_spec.rb

```diff
@@ -79,6 +79,15 @@
     end
   end
 
+  context "with related objects" do
+    subject(:work) { FactoryBot.create(:distinct_cytoskeletal_proteins_work) }
+    it "has related objects" do
+      expect(work.resource.related_objects.first.related_identifier).to eq "https://www.biorxiv.org/content/10.1101/545517v1"
+      expect(work.resource.related_objects.first.related_identifier_type).to eq "arXiv"
+      expect(work.resource.related_objects.first.relation_type).to eq "IsCitedBy"
```

Comment:
below I see `isCitedBy`... is it case insensitive, or is something not being tested? or we capitalize it for display on the page?

Reply:
We're going to be working with a controlled vocabulary and I don't actually know which capitalization option is correct. I was going to tackle that next.